README.md

```diff
@@ -7,6 +7,10 @@ A collection of commonly used pieces of Angular-related code that we use everyda
 
 ## Installation
 
+```bash
+ng add @infinum/ngx-nuts-and-bolts
```

Comment:
Please add instructions on how to add it to a nx workspace as well, similar to transloco docs